### PR TITLE
feat(faq): Adds information on noisy neighbors

### PR DESCRIPTION
**Pull request by thomastaylor312**
This is a commonly asked question by people looking to run wasmCloud in production so I figured it would be good to add here

### CODE REVIEW

File: docs/reference/faq.mdx [hunk 1]
```diff
@@ -102,6 +102,18 @@ wasmCloud is compatible with, but not dependent on, Kubernetes. wasmCloud hosts
...
+
+#### CPU/Memory Resource Contention
+
+The most common type of noisy neighbor problem is resource contention over things like CPU and memory. Because Wasm is an emerging technology there isn't a 100% standard way to handle this yet. However, we have plans to add resource scheduling to [Wadm](https://github.com/wasmcloud/wadm), our scheduling and orchestration tool. This will recognize requests and limits for actors and providers, and will ensure that resources are available for use, returning an error if things would be overprovisioned. Right now, you'll still have to manage enforcement of these limits at the OS or container level (which is currently the status quo for most things). However, in the future, because Wasm is extremely deterministic, we should be able to use a combination of the metrics a wasmCloud host exposes and knowledge of what steps a module will take to develop a much more flexible, dynamic, and automated way of handling limits and requests of resources.
```
Comment (connorsmith256) [inline docs/reference/faq.mdx]: Consider mentioning that while we don't yet have resource limits in terms of CPU/memory, we _do_ have limits in terms of concurrency, which can be a useful proxy. It's hard to be a noisy neighbor if only 10 actor instances can be active at the same time

**Review by connorsmith256 — APPROVED** [review]
LGTM. Once the production deploy section of the docs has been reworked, I think these will deserve their own page

**Review by CazJT — APPROVED** [review]

File: docs/reference/faq.mdx (hunk 1 above)
Comment (thomastaylor312) [reply docs/reference/faq.mdx]: Good point, I've added it

File: docs/reference/faq.mdx [hunk 2]
```diff
@@ -102,6 +102,18 @@ wasmCloud is compatible with, but not dependent on, Kubernetes. wasmCloud hosts
...
+
+#### CPU/Memory Resource Contention
+
+The most common type of noisy neighbor problem is resource contention over things like CPU and memory. Because Wasm is an emerging technology there isn't a 100% standard way to handle this yet. However, we have plans to add resource management to [Wadm](https://github.com/wasmcloud/wadm), our scheduling and orchestration tool. This will recognize requests and limits for actors and providers, and will ensure that resources are available for use, returning an error if things would be overprovisioned. Right now, we already have a `max_concurrency` value you can set for actors (when you start them manually or via Wadm), which will get you pretty far. But as for actually enforcing specific limits, you'll still have to manage enforcement of these limits at the OS or container level (which is currently the status quo for most things). However, in the future, because Wasm is extremely deterministic, we should be able to use a combination of the metrics a wasmCloud host exposes and knowledge of what steps a module will take to develop a much more flexible, dynamic, and automated way of handling limits and requests of resources.
```
Comment (rajsite) [inline docs/reference/faq.mdx:111]: > However, we have plans to add resource management to wadm, our scheduling and orchestration tool.

@thomastaylor312 are there issues / roadmaps that cover this? I was recently trying to get some folks to evaluate wasm cloud from a thinking outside the box / looking at alternatives perspective and the lack of specifics on this topic became an immediate no-go compared to their alternate architecture.

Comment (brooksmtownsend) [reply docs/reference/faq.mdx:111]: Hey @rajsite, so I actually pushed up part of the implementation for components in https://github.com/wasmCloud/wasmCloud/compare/main...feat/component-ram-limits which I should be working on this week, but may land in September. Automatically determining resource limits is a hard problem, but adding support to wadm&wasmCloud for specifying resource limits will land fairly soon. I'm happy to put up a couple issues around what I'm thinking will be covered in the implementation if that will help the research 😄

**Pull request merged**